Feat : 음원 상세페이지 - 음원 정보, 리뷰 작성/수정/삭제

hyunjooooojung opened the pull request:
## PR 타입(하나 이상의 PR 타입을 선택해주세요)
- [ ] 디자인 변경
- [x] 기능 추가
- [ ] 기능 삭제
- [ ] 버그 수정
- [ ] 의존성, 환경 변수, 빌드 관련 코드 업데이트

## 반영 브랜치
feature/feature/27-music-review-api -> API

## 변경 사항

1. 음원 상세 페이지에서 음원의 상세 정보를 불러온다. (handleMock())
    - [x] 음원 정보 불러오기
    - [x] 이 음원에 작성된 리뷰 목록 불러오기
    - [x] 테스트 결과
![image](https://user-images.githubusercontent.com/113044908/200255319-3d6d2878-bfa4-4021-9272-2e13df0b4935.png)

2. 음원 상세 페이지에서 리뷰를 작성, 수정, 삭제가 가능하다.
    - [x] 리뷰 작성(handleCreateReview()), 리뷰 수정(handleUpdateReview()), 리뷰 삭제(handleDeleteReview())
    - [x] 리뷰 수정, 삭제는 리뷰 작성자만 가능하다.
    - [x] 리뷰 수정을 위해 모달창을 만들었고 , 이 모달창에서 리뷰 수정이 가능하다.
    <img width="959" alt="Screenshot 2022-11-07 at 4 57 45 PM" src="https://user-images.githubusercontent.com/113044908/200256056-aad782a2-df66-4613-9ee6-aa4b650a6783.png">
    
    - [x] 테스트 결과
<img width="795" alt="Screenshot 2022-11-07 at 4 54 16 PM" src="https://user-images.githubusercontent.com/113044908/200255496-72cadb30-3bce-4981-bd7a-f9844331979b.png">

hyunjooooojung committed:
Merge branch 'API' of https://github.com/sparta-USD/Na-dle-Front into feature/27-music-review-api

ssuin52 reviewed (approved):
진짜진짜 고생 많으셨습니다ㅜㅡㅜ!! 굳이에요 굳!!! 굳굳굳!!!!!!!

hyunjooooojung committed:
Feat : 음원 상세정보 불러오기, 리뷰 작성/수정/삭제 #27

- 음원 상세페이지에서 음원의 상세정보를 불러온다.
- 상세 페이지에서 리뷰를 작성할 수 있다.
- 리뷰 수정, 삭제는 리뷰 작성자만 가능함.

Close #27

LeeHyunji reviewed (approved):
좋습니다. 다음에는 first_like.html 같이 다른 기능은 분리해주세요.

LeeHyunji committed:
Merge branch 'API' into feature/27-music-review-api
